blockchain/skipchain/member.go

```diff
@@ -48,25 +49,76 @@ func (c Conode) Pack() (proto.Message, error) {
 	return conode, nil
 }
 
+type iterator struct {
+	conodes []Conode
+	index   int
+}
+
+func (i *iterator) Next() bool {
```

Comment:
~Next~ HasNext should only check `if i.index < len(i.conodes)` and ~Get~ GetNext should return the conode AND increment the index.